web/views/api/index_view.ex

```diff
@@ -0,0 +1,8 @@
+defmodule HexWeb.API.IndexView do
+  use HexWeb.Web, :view
+
+  def render("index." <> _format, _assigns) do
+    %{packages_url: api_package_url(Endpoint, :index),
+      documentation_url: "http://docs.hexpm.apiary.io"}
+  end
```

Comment:
That's a great idea. I was thinking about how to make the non-index resources discoverable.

We should add keys even though they require auth.